Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM openjdk:8u121
+FROM ubuntu:16.04
 
 ARG MESOS_VERSION=1.6.0
 
-RUN touch /usr/local/bin/systemctl && chmod +x /usr/local/bin/systemctl
-
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF \
- && echo "deb http://repos.mesosphere.io/debian jessie main" > /etc/apt/sources.list.d/mesosphere.list \
+RUN apt-get update \
+ && apt-get install -y openjdk-8-jdk \
```

Comment:
there's already an `apt-get install` block below. please move these lines there.
Also, there's no need to run `apt-get update` or `apt-get clean` more than once.

Reply:
There are still 2 `apt-get update`s. Please move new `apt-get install`s below the existing `apt-get -y update`